keyboards/levinson/keymaps/jyh/keymap.c

```diff
@@ -0,0 +1,232 @@
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Esc  |  A   |  S   |  D   |  F   |  G   |    |  H   |  J   |  K   |  L   |  ;   |  '   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |  -   |  Z   |  X   |  C   |  V   |  B   |    |  N   |  M   |  ,   |  .   |  /   | Enter|
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Numpd| Meh  | Meta | Alt  | Lower| Space|    | Space| Raise| Alt  | Meta | Hyper| Func |
+ * `-----------------------------------------'    `-----------------------------------------'
+ *
+ * with Hold (Emacs-oriented)
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * | Hyper|      |      |      |      |      |    |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Ctrl |      |      |      |      |      |    |      |      |      |      |      | Ctrl |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * | Shift|      |      |      |      |      |    |      |      |      |      |      | Shift|
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_QWERTY] = KEYMAP(
+								HPR_TAB, KC_Q   , KC_W   , KC_E   , KC_R , KC_T   ,    KC_Y   , KC_U , KC_I   , KC_O   , KC_P   , KC_BSPC,
+								CTL_ESC, KC_A   , KC_S   , KC_D   , KC_F , KC_G   ,    KC_H   , KC_J , KC_K   , KC_L   , KC_SCLN, CTL_QOT,
+								SFT_MIN, KC_Z   , KC_X   , KC_C   , KC_V , KC_B   ,    KC_N   , KC_M , KC_COMM, KC_DOT , KC_SLSH, SFT_ENT,
+								NUMPAD , KC_MEH , KC_LGUI, KC_LALT, LOWER, CTL_SPC,    KC_SPC, RAISE, KC_RALT, KC_RGUI, KC_HYPR, MO(_FUNC)
+								),
+
+
+/* Raise
+ * ,-----------------------------------------.    ,-----------------------------------------.
+ * |  `   |  1   |  2   |  3   |  4   |  5   |    |  6   |  7   |  8   |  9   |  0   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      | Left |  Up  | Down | Right|  (   |    |  )   |  -   |  =   |  [   |  ]   |  |   |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |  ;   |  '   |  *   |  ,   |  {   |    |  }   |  _   |  +   |  .   |  /   |      |
+ * |------+------+------+------+------+------|    |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |    |      |      |      |      |      |      |
+ * `-----------------------------------------'    `-----------------------------------------'
+ */
+				[_RAISE] = KC_KEYMAP(
```

Comment:
`KC_KEYMAP` is no longer used for this keyboard

Please replace this with `LAYOUT_kc` or add KC_ to all of the keys and use  `LAYOUT_ortho_4x12`

```suggestion
				[_RAISE] = LAYOUT_kc(
```

```suggestion
				[_RAISE] = LAYOUT_ortho_4x12(
```